### DISCUSSION TITLE
[SOLVED][FR][tt-rss sync] "Published" feed

### DISCUSSION THREAD

**Discussion by uriel1998** [martinrotter/rssguard#647] [Ideas]
While the "importance" marker in RSSGuard maps to the "starred" flag in TT-RSS, there is no equivalent for the "published" flag.  Can there be a way to mark this in RSSGuard that syncs back to TT-RSS?  I use this functionality to have it automatically get articles archived to Wallabag and Shaarli.  Thanks!

**Comment by martinrotter** [comment]
Did you try latest development build? I already added support for "published" feed and also "post to published" feature.

![image](https://user-images.githubusercontent.com/1255302/154991356-51462917-ea59-4f1f-befa-e64467ed1041.png)

![image](https://user-images.githubusercontent.com/1255302/154991461-0f17a1d2-ddcb-40b1-a0da-2931ba241ca1.png)

![image](https://user-images.githubusercontent.com/1255302/154991507-665db7bc-dc34-4928-9bd1-27580d62dc9b.png)

Is that it?

**Reply by uriel1998** [reply]
Oh, no I had not.  (Running Debian bullseye/bookworm, and went with the AppImage instead of building from source.)  I'll take a look at that later this evening, and if it's the same thing, I'll mark this as *solved*.  Thanks!

**Comment by martinrotter** [comment]
There is testing appimage!

See main readme for link

**Comment by uriel1998** [comment]
Am I missing something?

![20222521-172544-](https://user-images.githubusercontent.com/1130750/155033451-44bd3296-e229-42a9-a9cd-cc892ff847f7.png)

**Reply by martinrotter** [reply]
No, there was build error for Linux.

It is rebuilding now and you should see appimage there soon in 30 minutes.

**Comment by uriel1998** [comment]
Got it!  Thanks! 

Note for documentation:  It took refreshing the folder list for the label to appear, which I didn't know to do originally but figured out. After that it showed up and let me add the label. That functionality worked perfectly. 

Note for development: Because there's not an icon for it, it's now made the small (not customizable) toolbar with "mark read" and "switch importance" really wide because of the length of the label (screenshot below). This was regardless of icon theme (I tried six or seven, including "hicolor" and "default".  Is there an icon name it's specifically looking for?
![20223922-073914-](https://user-images.githubusercontent.com/1130750/155134451-ad4575f4-015c-4dfa-955f-805f401da945.png)

**Reply by martinrotter** [reply]
This vertical toolbar displays list of labels/tags which can be assigned to article. It display color circle (color of the label) and title of label.

Indeed, if the title of label is too long, then the toolbar is just too wide. Let me check this.

**Reply by martinrotter** [reply]
I made a tweak, now the style of those label buttons in that toolbar follows rest of application, by default this should now only "icon" (colored circle in this case).

![image](https://user-images.githubusercontent.com/1255302/155138808-55fcffd4-f2eb-4c3d-861f-da80c2c74893.png)

![image](https://user-images.githubusercontent.com/1255302/155139016-f46dbaa0-7694-497a-8471-f7ba862802fb.png)

970dfc0c4

**Reply by uriel1998** [reply]
Awesome!!!

Marking [solved]
